integration/cluster.go

```diff
@@ -439,7 +441,8 @@ type member struct {
 	// PeerTLSInfo enables peer TLS when set
 	PeerTLSInfo *transport.TLSInfo
 	// ClientTLSInfo enables client TLS when set
-	ClientTLSInfo *transport.TLSInfo
+	ClientTLSInfo  *transport.TLSInfo
+	clientAutoSync time.Duration
```

Comment:
I don't think this option belongs in any of the cluster.go structures since it's client configuration, not server. Just create an autosyncing client directly in the test case instead of adding a new config option?